ffi/dotnet/Devolutions.IronRdp/Generated/RawPduHint.cs

```diff
@@ -17,7 +17,7 @@ public partial struct PduHint
     private const string NativeLib = "DevolutionsIronRdp";
 
     [DllImport(NativeLib, CallingConvention = CallingConvention.Cdecl, EntryPoint = "PduHint_find_size", ExactSpelling = true)]
-    public static unsafe extern ConnectorFfiResultBoxOptionalUsizeBoxIronRdpError FindSize(PduHint* self, byte* bytes, nuint bytesSz);
+    public static unsafe extern ConnectorFfiResultBoxOptionalUsizeBoxIronRdpError FindSize(PduHint* self, byte* bytes, nuint bytesSz, bool* matched);
```

Comment:
issue: Can you revert this for now? At least the C# NuGet is only used on client-side so we probably don’t need to break the ABI for that just now.

Reply:
ok